packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -144,8 +145,17 @@ - (void)start
     return;
   }
 
+  NSError *buildDataError;
+  [EXUpdatesBuildData ensureBuildDataIsConsistent:_database config:_config error:&buildDataError];
+  if (buildDataError) {
+    @throw [NSException exceptionWithName:NSInternalInconsistencyException
+                                   reason:buildDataError.localizedDescription
+                                 userInfo:buildDataError.userInfo];
```

Comment:
curious in general about your thoughts on how these errors are handled in this PR.